src/transformers/modeling_tf_t5.py

```diff
@@ -383,14 +383,21 @@ def _prune_heads(self, heads_to_prune):
 
     def call(
         self,
-        hidden_states,
+        inputs,
         attention_mask=None,
         encoder_hidden_states=None,
         encoder_attention_mask=None,
         head_mask=None,
         training=False,
     ):
-
+        if isinstance(inputs, (tuple, list)):
+            hidden_states = inputs[0]
```

Comment:
Nice, so normally we have all the inputs possible in the list/dict, i.e. here also attention_mask, encoder_hidden_states, encoder_attention_mask, head_mask see e.g. https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_tf_bert.py#L506-L523

The reason we do that is that it let you use all the features of Keras.fit() like type casting and so on, which are only implemented for the first positional input of the model

Reply:
Do you think you can add all the inputs here as well like in the example I linked to above.

Reply:
Actually the PR shouldn't be touching this class at all, as I backed out of making it Keras-serializable. Will remove.

Reply:
Removed.